test/unit/factories/Axis.mocha.ts

```diff
@@ -0,0 +1,40 @@
+/// <reference path='../test.mocha.ts' />
+
+describe('n3Charts.Factory.Axis', () => {
+  var domElement: JQuery = angular.element(document.body).append('<div></div>');
+  var axis: n3Charts.Factory.Axis = undefined;
+  var eventMgr: n3Charts.Utils.EventManager = undefined;
+  var factoryMgr: n3Charts.Utils.FactoryManager = undefined;
```

Comment:
Also, I was thinking about having a generic test we run on each class that inherits from BaseFactory, sort of a test utils function that would accept a class as an argument and run a bunch of tests to make sure the lifecycle runs the same for everyone